bindings/uniffi/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license-file.workspace = true
 thiserror = { workspace = true }
 web5 = { path = "../../crates/web5" }
 uniffi = { version = "0.27.1", features = ["cli"] }
+web5_uniffi_wrapper = { path = "../uniffi-wrapper" }
```

Comment:
I noticed you have an `_` above for the package name but a `-` for the file name

Reply:
Good catch, I don't like that inconsistency, I'll fix. Also I'm going to make the directory name match the exact rust crate name. So:

- `bindings/uniffi` --> `bindings/web5_uniffi`
- `bindings/uniffi-wrapper` --> `bindings/web5_uniffi_wrapper`

Reply:
done